pimd/pim_ifchannel.c

```diff
@@ -1354,15 +1335,11 @@ void pim_ifchannel_update_could_assert(struct pim_ifchannel *ch)
 	if (new_couldassert == old_couldassert)
 		return;
 
-	if (PIM_DEBUG_PIM_EVENTS) {
-		char src_str[INET_ADDRSTRLEN];
-		char grp_str[INET_ADDRSTRLEN];
-		pim_inet4_dump("<src?>", ch->sg.src, src_str, sizeof(src_str));
-		pim_inet4_dump("<grp?>", ch->sg.grp, grp_str, sizeof(grp_str));
-		zlog_debug("%s: CouldAssert(%s,%s,%s) changed from %d to %d",
-			   __func__, src_str, grp_str, ch->interface->name,
-			   old_couldassert, new_couldassert);
-	}
+	if (PIM_DEBUG_PIM_EVENTS)
+		zlog_debug("%s: CouldAssert(%pPAs,%pPAs,%s) changed from %d to %d",
```

Comment:
Here for printing group, we can keep as %pPA, s is not required here.

Reply:
These commits are intended to do 1:1 exact changes, the old behavior matches `%pPAs`. It can be changed to `%pPA` separately.